packages/account/src/Sections/Security/LoginHistory/login-history.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { Loading, ThemedScrollbars } from '@deriv/components';
+import { observer, useStore } from '@deriv/stores';
+import { WS } from '@deriv/shared';
+import LoadErrorMessage from 'Components/load-error-message';
+import LoginHistoryContent from './login-history-content';
+import { getLoginHistoryFormattedData } from '../../../../../utils/src/getLoginHistoryFormattedData';
+
+type TData = { id: number; date: string; action: string; browser: string; ip: string; status: string }[];
```

Comment:
*NIT*: May be u can call it `TLoginData` ? Generic names make it hard to understand